src/shared/varlink-io.systemd.Network.c

```diff
@@ -45,6 +46,22 @@ static SD_VARLINK_DEFINE_METHOD(
                 SetPersistentStorage,
                 SD_VARLINK_DEFINE_INPUT(Ready, SD_VARLINK_BOOL, 0));
 
+static SD_VARLINK_DEFINE_METHOD(
+                SetLinkDown,
+                SD_VARLINK_FIELD_COMMENT("Kernel interface index. If specified together with InterfaceName, both must reference the same link."),
```

Comment:
`Kernel` sounds spurious. I think simply `Interface index.` or `Index of the interface.` should be enough.